src/modules/dashboard/components/ExpenditurePage/Payments/Payments.tsx

```diff
@@ -0,0 +1,159 @@
+import React, { useCallback } from 'react';
+import { defineMessages, FormattedMessage } from 'react-intl';
+import { FieldArray, useField } from 'formik';
+import { useParams } from 'react-router';
+import { nanoid } from 'nanoid';
+import classNames from 'classnames';
+
+import Button from '~core/Button';
+import Recipient from '../Recipient';
+
+import styles from './Payments.css';
+import Icon from '~core/Icon';
+import { FormSection } from '~core/Fields';
+import { newRecipient } from './constants';
+import {
+  useColonyFromNameQuery,
+  useMembersSubscription,
+} from '~data/generated';
+import { SpinnerLoader } from '~core/Preloaders';
+
+const MSG = defineMessages({
+  payments: {
+    id: 'dashboard.Expenditures.Payments.defaultPayment',
```

Comment:
```suggestion
    id: 'dashboard.ExpenditurePage.Payments.defaultPayment',
```

This applies to all other message descriptor id's on this page